phpBB/assets/javascript/installer.js

```diff
@@ -78,12 +80,52 @@
 	}
 
 	/**
+	 * Render a download box
+	 */
+	function addDownloadBox(downloadArray)
+	{
+		var $downloadContainer = $('#download-wrapper');
+		var $downloadBox, $title, $content, $link;
+
+		for (var i = 0; i < downloadArray.length; i++) {
+			$downloadBox = $('<div />');
+			$downloadBox.addClass('download-box');
+
+			$title = $('<strong />');
+			$title.text(downloadArray[i].title);
+			$downloadBox.append($title);
+
+			if (downloadArray[i].hasOwnProperty('msg')) {
```

Comment:
Would be better to just check `downloadArray[i].msg` instead of using `.hasOwnProperty()`. Doesn't really matter that much, though.